website/static/js/pages/discussions-page.js

```diff
@@ -0,0 +1,16 @@
+/** Initialization code for the project discussion page. */
+var $ = require('jquery');
+var Comment = require('../comment.js');
+
+// Initialize comment pane w/ it's viewmodel
+var userName = window.contextVars.currentUser.name;
+var canComment = window.contextVars.currentUser.canComment;
+var hasChildren = window.contextVars.node.hasChildren;
+var target = window.contextVars.comment_target;
+target = (target==='None')? 'node' : target;
```

Comment:
I re-assign 'target' only when it is 'None'
However you are correct in that I am not correctly jsonifying 'comment_target' Will make the change in my next commit